organizational/conduct.rst

```diff
@@ -0,0 +1,101 @@
+building community, and promoting equality, Hacker Society must be a place where
+people can be comfortable regardless of gender, gender identity and expression,
+sexual orientation, disability, physical appearance, body size, race, age, or
+religion. Therefore, we do not tolerate:
+
+- Rude or unwelcome comments about a person's gender, gender identity, gender
+  expression, sexual orientation, disability, physical appearance, attire, race,
+  age or religious expression.
+- Sexual images in public spaces
+- Deliberate intimidation, stalking, or following
+- Harassing photography or recording
+- Sustained disruption of talks or other events
+- Inappropriate physical contact
+- Unwelcome sexual attention
+- Advocating for, or encouraging, any of the above behavior
+
+Necessarily, the goals of learning, community, and equality are in tension
+with the goals individual empowerment, freedom of thought, and freedom of
+expression. We strive for politeness and civil discourse but we also strive to
+make all welcome. Even those with unpopular ideas or thoughts. However, in
+order to foster our community certain limits on free expression must be
+observed. We expect that debate and constructive criticism are part of this
+open atmosphere. We expect that some members of the community may express
+ideas which are not politically correct. We expect that some ideas expressed
+by a minority may be deeply unpopular and distasteful to the majority.
+However, if these ideas or discussions devolve into attacks against a person
+and/or their intrinsic characteristics, we believe that this is unacceptable.
+
+Enforcement
+-----------
+
+This policy applies at all Hacker Society events and spaces, online and
+offline.  This includes, but is not limited to:
+
+- Weekly talks
+- Open hacks
+- Link-State
+- HackCWRU
+- Mailing lists (hackers-discuss, cwru-hackers)
+- GitHub organization
+- Slack team
+
+The policy applies to attendees as well as any speakers, sponsors, or other
+participants in these activities and communication streams.  Although we don't
+have any direct authority over behavior in our typical spaces such as the
+Glennan Student Lounge, we expect that HacSoc participants will adhere to these
+guidelines, and we also note that `Case's policies
+<https://students.case.edu/handbook/policy/>`_ are always in force.
+
+We expect participants asked to stop any harassing behavior to comply
+immediately.  If participants do not comply, the Maintainer and HacSoc officers
+retain the right to take any actions to keep HacSoc a welcoming environment for
+all participants.  This includes warning the offender, or removing them from the
+event or communication streams they are abusing.  This also includes further
+measures, such as pursuing University action, should it become necessary.
+
+Reporting
+---------
+
+If someone is making you feel unsafe or unwelcome at HacSoc, please report it as
```

Comment:
Suggested change:

> If you don't feel comfortable reporting a problem to an officer (perhaps because an officer is the problem) consider reporting the problem directly to any of the faculty or staff of the EECS department.